tests/test_automation/github-actions/ci/run_step.sh

```diff
@@ -41,7 +41,7 @@ case "$1" in
         export CUDACXX=/usr/local/cuda-11.2/bin/nvcc
 
         # Make current environment variables available to subsequent steps
-        echo "PATH=/usr/local/cuda-11.2/bin:$PATH" >> $GITHUB_ENV
```

Comment:
This might make the other Complex test fail.

Reply:
the old line caused PATH with "/usr/local/cuda-11.2/bin" being added twice.